src/app/services/graph-constants.ts

```diff
@@ -1,8 +1,15 @@
 export const GRAPH_URL = 'https://graph.microsoft.com';
 export const USER_INFO_URL = `${GRAPH_URL}/v1.0/me`;
+export const USER_BETA_INFO_URL = `${GRAPH_URL}/beta/me/profile`;
 export const USER_PICTURE_URL = `${GRAPH_URL}/beta/me/photo/$value`;
 export const AUTH_URL = 'https://login.microsoftonline.com';
 export const DEFAULT_USER_SCOPES = 'openid profile User.Read';
 export const DEVX_API_URL = 'https://graphexplorerapi.azurewebsites.net';
 export const GRAPH_API_SANDBOX_URL = 'https://proxy.apisandbox.msdn.microsoft.com';
 export const HOME_ACCOUNT_KEY = 'fbf1ecbe-27ab-42d7-96d4-3e6b03682ee4';
+export const APPLICATION = "Application";
+export const WORK = "DelegatedWork";
+export const PERSONAL = "DelegatedPersonal";
```

Comment:
could these constant names be more descriptive? we're talking about perm scope modes here right, what do you think about PERM_SCOPE_APPLICATION, PERMS_SCOPE_WORK?

Reply:
i also noticed PERSONAL isn't used in this PR (and delegated personal isn't supported by MS Graph iirc), would you agree that we should avoid adding unused code even if it's a constant that matches the rest of the set?

Reply:
yeah i just added it for sake of matching the set but it isn't used and might confuse people. i'll remove it

Reply:
after discussion in sprint grooming, i think ill leave in personal because its on our backlog to support delegatedPersonal perms after the MVP.